examples/datacalc.py

```diff
@@ -93,8 +98,8 @@ def updateDB(self, content: dict):
         newDBs = set([""])
         for db in content.get("db", ()):
             if any(key not in db for key in ("name", "path")):
-                print(f"The message was ignored because "
-                        f"the database {db} has no such key; name or path.")
+                logger.info("The message was ignored because "
+                        "the database %s has no such key; name or path.", json.dumps(db))
```

Comment:
Please align the starting positions of the two lines of double quotation marks.

```python
logger.info("The message was ignored because "
            "the database %s has no such key; name or path.", json.dumps(db))
```